### ISSUE TITLE
Installation on PyCharm

### ISSUE THREAD

**Issue by Fourthought** [explosion/spaCy#4597]
Hi,

I've been attempting to install spaCy on Pycharm but with no luck.

## How to reproduce the problem
Installation using: `pip install -U spacy`

spaCy is working well in Jupyter notebook, just now need to transfer to Pycharm for handling a large set of code.

The installation seems to fail at the point of installing Blis. After looking through previous issues both on Github and Stackoverflow it seems this is a 32bit to 64bit problem. I don't seem to be able to create a win64 venv in pycharm (I realise this may be out of scope for a spaCy installation problem)

Have been trying to sort this for three solid days now, so any assistance would be greatly appreciated!!

```bash
    Failed building wheel for blis
    Running setup.py clean for blis
    Building wheel for srsly (setup.py): started
    Building wheel for srsly (setup.py): finished with status 'done'
    Stored in directory: C:\Users\Steve\AppData\Local\pip\Cache\wheels\20\f7\c0\da1749f9bb8fc18fd8fcaadf0b2a420fd27d80a6e269025320
    Building wheel for numpy (setup.py): started
    Building wheel for numpy (setup.py): still running...
    Building wheel for numpy (setup.py): still running...
    Building wheel for numpy (setup.py): still running...
    Building wheel for numpy (setup.py): finished with status 'done'
    Stored in directory: C:\Users\Steve\AppData\Local\pip\Cache\wheels\5e\e9\4b\dd5a8eb53e97dfcc1314eca9c6769edd3cad379d6644c1ad94
  Successfully built thinc preshed cython srsly numpy
  Failed to build blis
  Installing collected packages: murmurhash, cymem, preshed, numpy, blis, wasabi, srsly, plac, tqdm, thinc, wheel, cython
    Running setup.py install for blis: started
      Running setup.py install for blis: finished with status 'error'
```

## Your Environment

* IDE: Pycharm
* Data from the Python Console within Pycharm
```bash
Python 3.8.0 (tags/v3.8.0:fa919fd, Oct 14 2019, 19:37:50) [MSC v.1916 64 bit (AMD64)] on win32
>>> import platform
>>> print(platform.architecture())
('64bit', 'WindowsPE')
```
* Operating System: windows 10
* Python Version Used: 3.8
* spaCy Version Used: 2.2.2
* Environment Information: while the IDE is the 64 bit installation, it seems to have provided win32.

**Comment by adrianeboyd** [comment]
For now, please use python 3.7 with windows. A more detailed explanation from Ines: https://github.com/explosion/spaCy/issues/4581#issuecomment-549336039

**Comment by Fourthought** [comment]
That's now working, thank you so much

**Comment by pramad11** [comment]
had similar issue which is fixed by installing 3.7.

I had 3.6.
